docs/integration-guide.md

```
@@ -0,0 +1,1132 @@
+You can obtain the DCQL query from a validated `AuthorizationDcqlRequest` and use it to identify matching credentials before prompting the user for selection and consent.
+
+### `DCQLHelper`
+
+`DCQLHelper` provides functionality for evaluating Wallet credentials against a DCQL query.
+
+#### Parameters
+
+| Name             | Type                      | Required | Default Value | Description                                                                                                                                                                                              |
+|------------------|---------------------------|:--------:|:-------------:|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `jsonLdExpander` | `JsonLdExpanderCallback?` |    No    |     `nil`     | Optional callback used to expand JSON-LD credentials before DCQL evaluation. This is required when the Wallet credentials are JSON-LD based as credential matching depends on expanded JSON-LD contexts. |
+|
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 5; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)